composer.json

```diff
@@ -13,12 +13,18 @@
     },
     "require": {
       "php": "7.*",
+      "ext-json": "*",
       "automattic/jetpack-connection": "1.30.13",
       "automattic/jetpack-config": "1.5.3",
       "automattic/jetpack-autoloader": "2.10.10",
       "myclabs/php-enum": "1.7.7",
-      "ext-json": "*",
-      "woocommerce/subscriptions-core": "1.1.0"
+      "woocommerce/subscriptions-core": "1.1.0",
+      "symfony/polyfill-php71": "1.20.0",
+      "symfony/polyfill-php72": "1.23.0",
+      "symfony/polyfill-php73": "1.23.0",
+      "symfony/polyfill-php74": "1.23.0",
+      "symfony/polyfill-php80": "1.23.1",
```

Comment:
In our current codebase state, polyfills for PHP 7.1 - 7.4 are not used, so we could drop those.

Reply:
I haven't double checked this, but trust you 🙂